sn_collections/_workshops/2024-ai-user-forum/20-a-ml-intro.md

```diff
@@ -0,0 +1,18 @@
+---
+title: An introduction to machine learning for science
+date: 2024-11-20 12:00
+categories: [2024 AI User Forum] 
+time: 1-4 pm CT
+lead: ARS SCINet Office
+
+section: November 20, Afternoon — Foundational Skills and Concepts
+
+prerequisites:
+  - text: Familiarity with basic command-line concepts. We will offer virtual training for these skills before the Forum begins.
```

Comment:
Prereq should be "Familiarity with basic Python concepts and Jupyter notebooks. We will offer virtual training for these skills before the Forum begins."